ui/desktop/electron-app/src/index.js

```diff
@@ -166,9 +167,13 @@ app.on('ready', async () => {
     /* eng-disable LIMIT_NAVIGATION_JS_CHECK */
     if (!url.startsWith('serve://boundary')) event.preventDefault();
   });
+  // Also opens links with target="_blank" in a browser window
   mainWindow.webContents.on('new-window', (event, url) => {
     /* eng-disable LIMIT_NAVIGATION_JS_CHECK */
-    if (!url.startsWith('serve://boundary')) event.preventDefault();
+    if (!url.startsWith('serve://boundary')) {
+      event.preventDefault();
+      shell.openExternal(url);
```

Comment:
Thoughts on adding a documentation url and main boundary project url (https://boundaryproject.io/ can be launched from menu option) check as a protection against opening unknown urls?

Reply:
Yeah I did not ask about this, but you are right.

Should we do an authorized list of domains? let's say we just open these externals domains documentation url and main boundary project? any more suggestions?

Reply:
I can't think of any additional domains to add. It'll be a good allow domain bucket to start with.

Reply:
@gsusmi Since we are just allowing one domain for now, I think is best to keep a simple implementation. With allow a domain bucket you mean to could admit an array of domains? In that case I will wait until we have to deal with more than 1 domain to change the implementation, but happy to change it if you don't think so.